### PR TITLE
Removed service worker logs and replaced them with console errors

### PR DESCRIPTION
**Pull request by anuran-roy**
Removed service worker logs and replaced them with console errors and replaced them with console errors and sentry reporting

### CODE REVIEW

File: src/Components/Notifications/NotificationsList.tsx [hunk 1]
```diff
@@ -65,9 +67,11 @@ export default function ResultList({ expanded = false, onClickCB }: Props) {
         setIsSubscribed("SubscribedOnAnotherDevice");
       }
     } catch (error) {
-      Error({
-        msg: `Service Worker Error - ${error}`,
-      });
+      // Error({
```
Comment (developedBySJ) [inline src/Components/Notifications/NotificationsList.tsx]: Clean up code

File: src/Components/Notifications/NotificationsList.tsx [hunk 2]
```diff
@@ -126,7 +130,9 @@ export default function ResultList({ expanded = false, onClickCB }: Props) {
           });
       })
       .catch(function (_e) {
-        Error({ msg: "Service Worker Error" });
+        // Error({ msg: "Service Worker Error" });
```
Comment (developedBySJ) [inline src/Components/Notifications/NotificationsList.tsx]: Clean up code

File: src/Components/Notifications/NotificationsList.tsx [hunk 3]
```diff
@@ -65,9 +67,11 @@ export default function ResultList({ expanded = false, onClickCB }: Props) {
         setIsSubscribed("SubscribedOnAnotherDevice");
       }
     } catch (error) {
-      Error({
-        msg: `Service Worker Error - ${error}`,
-      });
+      // Error({
+      //   msg: `Service Worker Error - ${error}`,
+      // });
+      console.log(`Service worker error...Details: ${error}`);
```
Comment (developedBySJ) [inline src/Components/Notifications/NotificationsList.tsx]: use `console.error`

**Review by developedBySJ — CHANGES_REQUESTED** [review]

**Comment by anuran-roy** [comment]
@developedBySJ can you please review it?

**Comment by anuran-roy** [comment]
@kunatastic can we remove the `need testing` label?

**Comment by kunatastic** [comment]
@anuran-roy ig @nihal467 would be the right person

**Comment by nihal467** [comment]
looks good

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**